### DISCUSSION TITLE
Usage of sync_list (for only syncing certain directories/files) in Docker.

### DISCUSSION THREAD

**Discussion by stelladraco27** [abraunegg/onedrive#2509] [Q&A] [answered]
Hey there.
As a bit of an intro, I am setting up a windows server, using Docker, to get a notes app service up and running called SilverBullet. I want this notes app to access/save/etc. its files to my OneDrive so I can access them elsewhere. The challenge is my OneDrive is close to a terabyte in size, and because a small folder is what my notes are stored in, I do not want to sync everything, else I'd just use the native OneDrive Sync application. I came across this project in hopes to be able to only sync this one directory, and after looking through the documentation I know it's possible through the `sync_list` file; however, I am not experienced in Docker enough to know how to edit the actual "onedrive" command to allow this. Can anyone help? Thank you in advance.

**Comment by abraunegg** [comment, marked as answer]
@stelladraco27 
Sorry to re-direct you, however will probably get a better answer and response to your question by asking it here: https://forums.docker.com/
